src/IniFile.h

```diff
@@ -114,4 +111,4 @@ class CIniFile
 			return Start.Key > End.Key;
 		}
 	};
-};
+};
```

Comment:
It's customary to have a newline at the end of the file, rather than finishing on the same line as code.